helix-core/src/indent.rs

```diff
@@ -346,6 +578,32 @@ mod test {
 
         let does_indentation_work = 1;
 
+        let mut really_long_variable_name_using_up_the_line =
```

Comment:
If calculation is good enough to match `cargo fmt` yet, we can `include_str!()` `.rs` files in this crate or `helix-term/src/commands.rs` and calculate indentation for the entire file. I was originally doing that but the formatter wasn't able to match `cargo fmt` in all cases.

Reply:
I've thought about this as well, the current solution with an inline string isn't ideal. Could we maybe have a `test-indent` folder somewhere for these test files? That way we could also test languages other than rust (some future features like `align` don't really apply to rust). For rust, we could still just add a symlink.

Reply:
Agreed!